jdbc/src/main/java/com/yahoo/ycsb/db/JdbcDBClient.java

```diff
@@ -195,7 +195,7 @@ public void init() throws DBException {
       }
       int shardCount = 0;
       conns = new ArrayList<Connection>(3);
-      final String[] urlArr = urls.split(",");
+      final String[] urlArr = urls.split(";");
```

Comment:
Please add a comment explaining what's going on here, syntactically.

Reply:
Sorry for coming back to this so late. 
In case a connection String with multiple hosts is provided it will split the String based on the delimiter ";" instead of using ",". 
This is required to enable as well the useage of the failover mechanisms of MySQL and PostgesSQL as the syntax for failover connection Strings are `jdbc:mysql/postgres://host1:port1,host2:port2,..`. 
So in case a sharding connection (as it is now) needs to be specified as follows: 
`jdbc:mysql/postgres://host1:port1;host2:port2;..`
A failover connection String as follows:
`jdbc:mysql/postgres://host1:port1,host2:port2,..`
And a sharded failover connection as follows:
`jdbc:mysql/postgres://host1:port1,host2:port2;host3:port3,host4:port4...`